src/molecules/form-fields/input-form-field.test.tsx

```diff
@@ -1,6 +1,10 @@
 import React from "react";
-import { render } from "@testing-library/react";
-import { InputFormField } from "./input-form-field";
+import { render, fireEvent } from "@testing-library/react";
+import {
+    InputFormField,
+    InvalidInputFormValueClass,
+    ShowLabelForScreenReadersOnlyClass,
```

Comment:
Could these two classnames be namespaced to the `InputFormField` component? something a little more descriptive but hopefully not too verbose - 

`InputFormFieldInvalidClass`
`InputFormFieldScreenReaderLabelClass`